pcsx2/GS/Renderers/Metal/GSTextureMTL.h

```diff
@@ -67,6 +68,12 @@ class GSTextureMTL : public GSTexture
 	void FlushClears();
 	/// Marks pending clears as done (e.g. if the whole texture is about to be overwritten)
 	void InvalidateClears();
+	/// Marks the texture as invalid (will load with DontCare)
```

Comment:
Rather than introducing a Metal-specific field, you can use the state in the base class (Dirty/Cleared/Invalidated), which is what I use in Vulkan/DX12.

Reply:
Done

I also got rid of stencil clearing as it seems we don't use it outside of individual renderers and GSTexture had nothing set up to handle it

Reply:
Yeah, it's only used for DATE, and always cleared to a fixed value.